### DISCUSSION TITLE
Add game history/recent games for all Battlesnakes on the Arena Leaderboard UI

### DISCUSSION THREAD

**Discussion by pedrosorio** [BattlesnakeOfficial/feedback#204] [Feature Requests]
A recent update to the website shows a new version of the leaderboard for the summer league:
https://play.battlesnake.com/league/summer-league-2022/arenas/summer-league-2022/

Which is significantly different from the previous one (still used in the battlegrounds):
https://play.battlesnake.com/ladder/summer-league-2022/

I really like that you can click any snake in the old UI and see the last round performance, as well as history + easily access a list of recent games played by this snake. 

In the new UI, clicking a snake just goes to the snake's page. Are there any plans to keep the useful functionality from the old UI in the new one?

**Comment by originalwebgurl** [comment]
Hi @pedrosorio,

This is currently being worked on. We'll be reintroducing it iteratively in some upcoming UI releases. 

The first iteration will allow you to see a table of the players most recent games and replay them (similar to what was added for your own snake). This is prioritized for our next development sprint. I'll let you know when it gets released so you can review it.

We're redesigning the history & charts into reusable components we can share between leagues, battlegrounds and any future ladders & leaderboards so they will take a bit longer to go live. Hopefully they'll be ready before Fall League. Again, I'll make sure to let you know when they're back as well. 

Thanks so much for opening this discussion. We love getting your feedback! If you have any suggestions or would like some sneak peeks at our mockups just let me know. I may be sharing some soon with our Community Council, but I don't mind sharing some here in this discussion thread as well.

**Reply by pedrosorio** [reply]
Thank you for the reply @originalwebgurl that sounds great!

**Comment by bvanvugt** [comment]
(moved from question to feature request, since this is being worked on)

**Comment by Llourn** [comment]
Heya @pedrosorio !
This has been added! 🎉 You can see it in action on the qualifying ladders right now. Thanks for the feedback! 👍🏻
